plugins/SitesManager/API.php

```diff
@@ -598,7 +598,7 @@ public function addSite($siteName,
         }
 
         $coreProperties = array();
-        $coreProperties = $this->setSettingValue('urls', $urls, $coreProperties, $settingValues);
+        $coreProperties = $this->setSettingValue('urls', $urls, $coreProperties, $settingValues, $isRequired = true);
```

Comment:
This makes it impossible to create any new roll ups, as for roll ups you can't define any urls.

Reply:
Good point. I guess I can make it required if type == 'website'. (cc @tsteur)